solutions/152/01.py

```
@@ -0,0 +1,18 @@
+def maxProduct(nums: List[int]) -> int:
```

Ruff (0.12.2)

Undefined name `List`

(F821)